### ISSUE TITLE
Multi line string issue

### ISSUE THREAD

**Issue by zhengbli** [Microsoft/TypeScript-TmLanguage#5]
When using multi line string in typescript, the code colouring is broken.
```typescript
let name = '\
Dirk \
Baeumer \
'
```

Ported from https://github.com/Microsoft/TypeScript-Sublime-Plugin/issues/392

**Comment by DanielRosenwasser** [comment]
We'd have to make sure that something like this:

```ts
let x = "abcd\\
"
```

doesn't get considered multiline

I wonder if this is just a matter of switching these from

```
  qstring-double:
    name: string.double.ts
    begin: '"'
    end: '"|(?=$)'
    patterns:
    - include: '#string-character-escape'
```

to

```ts
  qstring-double:
    name: string.double.ts
    begin: '"'
    end: '"|(?=(\\\\)+[^\]$)'
    patterns:
    - include: '#string-character-escape'
```

**Comment by zhengbli** [comment]
We do need to set up a test suite to test both cases. Though it might be tricky to set up CI for sublime text as UI manipulation is required. Maybe something can be done using atom / code

**Comment by zhengbli** [comment]
Fixed by #102
